content/tps/2025/2/individual.md

```diff
@@ -0,0 +1,661 @@
+---
+title: "20252C - Individual"
+subtitle: "Individual"
+---
+# Ingenieria de Software II
+## Trabajo Practico Individual
+
+### Entrega
+
+Gracias por tu interés en unirte a Melodia como Software Engineer.
+Estamos construyendo la próxima gran experiencia de streaming musical: intuitiva, envolvente y diseñada para que cada usuario descubra su nueva canción favorita en segundos. Queremos que formar parte de Melodia se sienta como diseñar el futuro de la música… porque eso es exactamente lo que harás.
+
+Tendrás 10 días para completar esta prueba técnica. Calculamos que llevará unas 4–6 horas, pero te damos total flexibilidad para que puedas lucir lo mejor de tu talento.
+
+Si surge cualquier pregunta sobre los requerimientos o el alcance, escríbenos: estamos aquí para que brilles y disfrutes el proceso tanto como los usuarios disfrutan su música.
+
+La entrega del proyecto se realizará mediante un repositorio privado en GitHub. Cada candidato deberá crear su propio repositorio y agregar a su corrector como colaborador para que pueda ser evaluado. Se recomienda iniciar tempranamente y hacer commits pequeños que agreguen funcionalidad incrementalmente. No se evaluará ningún commit realizado luego de la fecha y hora límite de entrega.
+
+⚠️ **Aviso importante:** Si durante la revisión de tu entrega surge alguna duda o inconsistencia, el equipo podrá solicitar una breve defensa oral a través de una reunión virtual (Google Meet u otra plataforma similar).
+
+El repositorio debe incluir un archivo `README.md` (en español) con:
+
+-   Una tabla de contenido.
+-   Una introducción con no más de un párrafo pequeño y conciso sobre la solución planteada.
+-   Una sección sobre qué fue lo más desafiante del proyecto.
+-   Un apartado de pre-requisitos listando lo necesario para levantar el entorno de desarrollo, especificando los lenguajes y versiones de los manejadores de paquetes necesarios.
+-   Link al "user-guide" de la libreria que se uso para testear, o en su defecto link al repo. e.g: [Junit](https://junit.org/junit5/docs/current/user-guide/), [gin-gonic](https://gin-gonic.com/docs/testing/)
+-   Comandos para construir la imagen de Docker.
+-   Comandos para correr la base de datos.
+-   Comandos para correr la imagen del servicio.
+
+Fecha máxima de entrega: [FECHA DE ENTREGA]
+
+
+#### Formulario de entrega 
+
+Se deberá completar el siguiente formulario con Padrón, Nombre, Apellido y Link al repositorio con la solución
+
+[Formulario]()
```

Comment:
```
[Formulario]()
[Formulario](https://forms.gle/vBRbDpQBYDzm1cd7A)
```
cc: @lucaslavandeira  I created the form for 20252C, it’s in our Drive. For now it’s set to “no acepta respuestas”. We must to enable it during the first class and let the students know